name-registry/tests/utils/mod.rs

```diff
@@ -0,0 +1,36 @@
+use fuels::{prelude::*, tx::ContractId};
+
+// Load abi from json
+abigen!(MyContract, "out/debug/name-registry-abi.json");
+
+pub async fn get_contract_instance() -> (MyContract, ContractId, WalletUnlocked, WalletUnlocked) {
+    // Launch a local network and deploy the contract
+    let mut wallets = launch_custom_provider_and_get_wallets(
+        WalletsConfig::new(
+            Some(2),             /* Single wallet */
+            Some(1),             /* Single coin (UTXO) */
+            Some(1_000_000_000), /* Amount per coin */
+        ),
+        None,
+    )
+    .await;
+    let wallet = wallets.pop().unwrap();
+    let wallet2 = wallets.pop().unwrap();
+
+    let id = Contract::deploy(
+        "./out/debug/name-registry.bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::with_storage_path(Some(
+            "./out/debug/name-registry-storage_slots.json".to_string(),
+        )),
+    )
+    .await
+    .unwrap();
+
+    let instance = MyContractBuilder::new(id.to_string(), wallet.clone()).build();
+
+    (instance, id.into(), wallet, wallet2)
+}
+
+pub mod abi;
```

Comment:
Move mod to the top.

Is there a reason for having a separate `abi` module rather than including it here? It's a matter of preference but I'm curious since the other apps follow the other pattern.

Reply:
Someone requested it as a change

Reply:
Correct, they did. That's because you've had all of them free-floating in the same file rather than like what the other apps do which is to have a mod inside the file for abi calls rather than move everything out of the file into a new module.